### ISSUE TITLE
Adding Gutenberg

### ISSUE THREAD

**Issue by Keats** [netlify/binrc#7]
Would it be ok for me to do a PR to add support for Gutenberg: https://github.com/Keats/gutenberg (https://www.getgutenberg.io)?

**Comment by calavera** [comment]
Absolutely!

If you don't want to change the code here, it might be faster for you to generate a tarball for your releases with the default format:

https://github.com/netlify/binrc/blob/master/cache/cache.go#L47

For example, for your v0.2.1 release, if you have a tarball called `Keats_gutenberg_Linux-64bit.tar.gz` and the binary inside is in this path `Keats_gutenberg_linux_amd64/Keats_gutenberg_linux_amd64`, you would have to do nothing for Netlify users to have access to Gutenberg today. They could pick versions with `GUTENBERG_VERSION=v0.2.1`.

Adding your format to the binary template is also ok, it might just take us some time to deploy a new version.

**Comment by Keats** [comment]
I assumed it would be something like adding a key on https://github.com/netlify/binrc/blob/master/templates/templates.toml for the format, I'd rather add the current format though.

I can't get the project working though:

```bash
~/Code/go/src/binrc(branch:master) » make deps                                                                                                                                                          vincent@arch
could not find project Gopkg.toml, use dep init to initiate a manifest
```

And running `dep init` fails as well:

```bash
No versions of github.com/netlify/binrc met constraints:
	v0.2.1: Could not introduce github.com/netlify/binrc@v0.2.1, as its subpackage github.com/netlify/binrc/statik is missing. (Package is required by (root).)
	v0.2.0: Could not introduce github.com/netlify/binrc@v0.2.0, as its subpackage github.com/netlify/binrc/statik is missing. (Package is required by (root).)
	v0.1.1: Could not introduce github.com/netlify/binrc@v0.1.1, as its subpackage github.com/netlify/binrc/statik is missing. (Package is required by (root).)
	v0.1.0: Could not introduce github.com/netlify/binrc@v0.1.0, as its subpackage github.com/netlify/binrc/statik is missing. (Package is required by (root).)
	master: Could not introduce github.com/netlify/binrc@master, as its subpackage github.com/netlify/binrc/statik is missing. (Package is required by (root).)
	osx-10: Could not introduce github.com/netlify/binrc@osx-10, as its subpackage github.com/netlify/binrc/statik is missing. (Package is required by (root).)

```

**Comment by calavera** [comment]
I'll take a look, I think it's because I used a very old version of `dep`, the dependencies manager, and they changed the file.

**Comment by calavera** [comment]
@Keats I just merged #9 in master, which fixes the build.

Feel free to open a PR with your changes to support gutenberg. We might be able to ship a new version next week.
